lib/Parsers/Convert.pm

```diff
@@ -0,0 +1,34 @@
+# SUSE's openQA tests
+
+# Copyright 2021 SUSE LLC
+# SPDX-License-Identifier: GPL-2.0-or-later
+
+package Parsers::Convert;
+
+use base Exporter;
+
+use strict;
+use warnings;
+
+use JSON;
+use YAML::Any;
```

Comment:
Is there a specific reason to use YAML::Any here, but for Parsers::YAML to use YAML::PP?
I think it would make more sense to be consistent.
The parsers YAML::Any supports behave all differently, and it doesn't support YAML::PP. Depending on what is installed you could get very different behaviour.
I don't recommend YAML::Any.
YAML::PP is the perl YAML module that is closest to the spec.